docs/en/guides/sre/s3-multi-region.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebar_label: Replicating a single shard across two AWS regions using S3 Object Storage
+---
+
+# Replicating a single shard across two AWS regions using S3 Object Storage
+
+- [Deploy ClickHouse](#deploy-clickhouse)
+- [Configure ClickHouse Keeper](#configure-clickhouse-keeper)
+- [Create S3 Buckets](#create-s3-buckets)
+- [Define a cluster](#define-a-cluster)
+- [Configure networking](#configure-networking)
+- [Testing](#testing)
+
+## Deploy ClickHouse
+
+Deploy ClickHouse on three hosts, in the sample configurations these are named `chnode1`, `chnode2`, and `keepernode1`.  `chnode1` and `chnode2` run ClickHouse and ClickHouse Keeper.  `keepernode1` is only running ClickHouse Keeper.
+
+Place `chnode1` in one AWS region, and `chnode2` in a second.  `keepernode1` can be placed in either of the two regions, maybe in a different availability zone within the region?
+
+## Configure ClickHouse Keeper
+
+ClickHouse Keeper is configured by adding a file to `/etc/clickhouse-server/config.d/` to override the default ClickHouse configuration.  All three servers use the same configuration with one setting different; `<server_id>` in the `<keeper_server>` section must match the `<server>` in the `<raft_configuration>`.  See the highlighted lines below.
```

Comment:
I (still) recommend running Keepers as standalone processes - it makes easier to upgrade or restart clickhouse-server.

Reply:
agreed, I need to update the main Keeper docs with the steps also.